sdk/python/packages/flet-core/src/flet_core/__init__.py

```diff
@@ -21,6 +21,9 @@
 from flet_core.audio import Audio
 from flet_core.badge import Badge
 from flet_core.banner import Banner
+from flet_core.menu_bar import MenuBar, MenuStyle
+from flet_core.submenu_button import SubmenuButton
+from flet_core.menu_item_button import MenuItemButton
 from flet_core.blur import Blur, BlurTileMode
```

Comment:
Please enable automatic sort of imports on save (https://github.com/flet-dev/flet/blob/main/CONTRIBUTING.md#sort-imports-on-save) or sort imports manually.